cypress/constants/selectors/selectors.js

```diff
@@ -0,0 +1,46 @@
+export const sigupSelectors =  {
+  email : '[data-cy=signup-email-text-field]',
+  submitEmail : '[data-cy=signup-email-submit-button]',
+};
+
+export const infoDetailsSelectors = {
```

Comment:
Instead of creating a single selector file, we can create separate selector files. Please check suggestions on the previous PR.